contract_manager/scripts/deploy_evm_pricefeed_contracts.ts

```diff
@@ -0,0 +1,298 @@
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import { EvmChain } from "../src/chains";
+import { DefaultStore } from "../src/store";
+import { existsSync, readFileSync, writeFileSync } from "fs";
+import {
+  DeploymentType,
+  EvmPriceFeedContract,
+  getDefaultDeploymentConfig,
+  PrivateKey,
+  toPrivateKey,
+  WormholeEvmContract,
+} from "../src";
+import { join } from "path";
+import Web3 from "web3";
+import { Contract } from "web3-eth-contract";
+
+type DeploymentConfig = {
+  type: DeploymentType;
+  validTimePeriodSeconds: number;
+  singleUpdateFeeInWei: number;
+  gasMultiplier: number;
+  gasPriceMultiplier: number;
+  privateKey: PrivateKey;
+  jsonOutputDir: string;
+  saveContract: boolean;
+};
+
+const CACHE_FILE = ".cache-deploy-evm";
+
+const parser = yargs(hideBin(process.argv))
+  .scriptName("deploy_evm_pricefeed_contracts.ts")
+  .usage(
+    "Usage: $0 --std-output-dir <path/to/std-output-dir/> --private-key <private-key> --chain <chain0> --chain <chain1>"
+  )
+  .options({
+    "std-output-dir": {
+      type: "string",
+      demandOption: true,
+      desc: "Path to the standard JSON output of the contracts (build artifact) directory",
+    },
+    "private-key": {
+      type: "string",
+      demandOption: true,
+      desc: "Private key to use for the deployment",
+    },
+    chain: {
+      type: "array",
+      demandOption: true,
+      desc: "Chain to upload the contract on. Can be one of the evm chains available in the store",
+    },
+    type: {
```

Comment:
Can remove this? Not being used below.

Reply:
Thanks for catching it. actually i wanted to use it 😅